bundles/org.openhab.persistence.influxdb/src/main/java/org/openhab/persistence/influxdb/internal/influx2/InfluxDB2RepositoryImpl.java

```diff
@@ -137,6 +145,42 @@ public boolean write(List<InfluxPoint> influxPoints) {
         return true;
     }
 
+    @Override
+    public boolean remove(FilterCriteria filter) {
+        final DeleteApi currentDeleteApi = deleteAPI;
+        if (currentDeleteApi == null) {
+            return false;
+        }
+
+        if (filter.getState() != null) {
+            logger.warn("Deleting by value is not supported in InfluxDB v2.");
+            return false;
+        }
+        OffsetDateTime start = Objects.requireNonNullElse(filter.getBeginDate(), ZonedDateTime.now().minusYears(100))
+                .toOffsetDateTime();
+        OffsetDateTime stop = Objects.requireNonNullElse(filter.getEndDate(), ZonedDateTime.now().plusYears(100))
+                .toOffsetDateTime();
```

Comment:
Is there a particular reason for using now +/- 100 years rather than `OffsetDateTime.MIN`/`OffsetDateTime.MAX`, for example? I should add that I don't know the internals of InfluxDB and the used API.

Reply:
That's a very good question. I just kept it the same way it is done for missing start/end dates in other queries.